Vagrantfile

```diff
@@ -29,6 +29,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision :shell, :inline => <<-EOS
     # To prevent "dpkg-preconfigure: unable to re-open stdin: No such file or directory" warnings
     export DEBIAN_FRONTEND=noninteractive
+    # Make sure git submodules are checked out!
+    apt-get -qq install -y git >/dev/null
+    git submodule --quiet update --init --recursive
```

Comment:
Sigh, sorry for wasting your time there, I thought when I pushed it that I had checked it earlier, but clearly I did not :-/ Thanks for coming up with the Vagrantfile; that'll break the submodule inside the guest, though :) Looking at http://stackoverflow.com/questions/17568543/git-add-doesnt-work along with https://github.com/git/git/commit/d75219b4a8a9c44520ddca234cde992498383b89 and https://github.com/git/git/commit/ea115a0d43d176f873855f9a7372376161dd8e65 the issue is the Vagrantfile uses a precise box, which is git 1.7.9; relative paths should still work, they were just made the default in 1.7.10, so switching to use them should let it work in both guest and host.